Libraries/LibJS/AST.cpp

```diff
@@ -1549,8 +1549,6 @@ Value ObjectExpression::execute(Interpreter& interpreter, GlobalObject& global_o
             continue;
         }
 
-        if (interpreter.exception())
-            return {};
```

Comment:
This is not needed - if we enter the if branch above, this is never reached, and otherwise there's an exception check right before that.